Metadata/SFBAudioMetadata.m

```diff
@@ -620,7 +620,7 @@ - (void)removeAttachedPicture:(SFBAttachedPicture *)picture
 
 - (void)removeAttachedPicturesOfType:(SFBAttachedPictureType)type
 {
-	for(SFBAttachedPicture *picture in _pictures) {
+	for(SFBAttachedPicture *picture in _pictures.copy) {
```

Comment:
Could you explain the reasoning behind this change? I don't think it's necessary.

Reply:
Deleting elements while looping through an array causes a crash

Reply:
What about something like:

```
	NSSet *picturesToRemove = [_pictures objectsPassingTest:^BOOL(SFBAttachedPicture *obj, BOOL *stop) {
#pragma unused(stop)
		return obj.pictureType == type;
	}];
	[_pictures minusSet:picturesToRemove];
```

Reply:
It's pretty good